scripts/analytics/run_keen_summaries.py

```diff
@@ -0,0 +1,98 @@
+import time
+import logging
+import argparse
+import importlib
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from keen.client import KeenClient
+
+from website.app import init_app
+from website.settings import KEEN as keen_settings
+from scripts.analytics.user_summary import get_events as user_events
+from scripts.analytics.node_summary import get_events as node_events
+from scripts.analytics.node_log_events import get_events as node_log_events
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events_for_day(day, scripts=None):
+    """Get an event summary for a certain 24 hour period.
+     00:00UTC - 23:59:59UTC on the day provided.
+
+     Default to explicitly imported and run analytic summary scripts,
+     but optionally pass in the names of specific scripts to run.
+     """
+    logger.info('<---- Keen Summary Counts for {} ---->'.format(day.isoformat()))
+
+    keen_events = {}
+    if scripts:
+        for script in scripts:
+            try:
+                script_events = importlib.import_module('scripts.analytics.{}'.format(script))
+                keen_events.update({script: script_events.get_events(day)})
+            except ImportError as e:
+                logger.error(e)
+                logger.error('Error importing script - make sure the script specified is inside of scripts/analytics')
+    else:
+        keen_events.update({'user_summary': user_events(day)})
+        keen_events.update({'node_summary': node_events(day)})
+        keen_events.update({'node_log_events': node_log_events(day)})
```

Comment:
Make the collection name an attribute of the script, so it's not hardcoded in two places.